main.tf

```diff
@@ -0,0 +1,54 @@
+variable "amis" {
```

Comment:
I know you only have one variable, but for convention and as a good practice in case the number of variables grows, I would suggest you to put this variable in a vars.tf file, and add a description, like this

variable "amis" {
    type = "map"
    description = "This variable contains amis for Virgina, Europe an Brasil"
}

when someone is using a terraform resource or module you wrote, it is really hard to find every variable required by your module, once you have your variables at vars.tf at the same level as main.tf I suggest you to use [terraform-docs](https://github.com/segmentio/terraform-docs) to generate a Markdown file, that way your terraform will auto document itself.

Reply:
Nice!.  I moved the variable to the new vars.tf file. Thanks for the suggestion.
Also will install terraform-docs, and add the generated files to the Pull request. I will let you know when they are ready.